includes/class-wc-siftscience-events.php

```diff
@@ -142,6 +142,14 @@ public function update_order_status( $order_id ) {
 			$this->send_transaction( $order_id );
 		}
 
+		private function is_auto_send($order_id) {
+			$min_value	= $this->get_min_order_value();
```

Comment:
Did you test this code? $this->get_min_order_value() doesn't exist.

Reply:
couldn't test this particular because this could send data to sift 
maybe this builds cost -i don't know 
i was about to tell you test sift 
--sorry for inconvenience

Reply:
No, using sift is free (sandbox mode is free).
Also, no:  `$this->get_min_order_value()` doesn't work. It should be `$this->options->get_min_order_value()`